server.go

```diff
@@ -360,6 +360,7 @@ func MaxSendMsgSize(m int) ServerOption {
 
 // MaxConcurrentStreams returns a ServerOption that will apply a limit on the number
 // of concurrent streams to each ServerTransport.
+// If this is not set, gRPC uses the default `math.MaxInt32`.
```

Comment:
Wouldn't this be `math.MaxUint32`?  But it might be nicer to just say "If this is not set, there is no limit."